python/common/python/math/__init__.java

```diff
@@ -0,0 +1,840 @@
+package python.math;
+
+import java.math.BigInteger;
+import java.math.BigDecimal;
+import python.errorfunction.math_erf;
+import python.gammafunction.math_gamma;
+
```

Comment:
As discussed, it's necessary to insert a credit to the original Jython source, since it's definitely derived from it. I think it'd be best to both have an attribution statement, and also either a reference or a direct link to their repo of the specific version/file that is the source.

This should be done for the erf/gamma files as well.